__tests__/app.test.js

```diff
@@ -85,26 +85,120 @@ describe("GET /api/reviews/:review_id", () => {
     return request(app)
       .get("/api/reviews/10000")
       .expect(404)
-      .then(({ body : {msg} }) => {
-        expect(msg).toBe("not found")
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("not found");
       });
   });
   test("400: bad request when review_id is an invalid data type", () => {
     return request(app)
       .get("/api/reviews/pineapple")
       .expect(400)
-      .then(({ body : {msg} }) => {
-        expect(msg).toBe("bad request")
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("bad request");
       });
   });
 });
 
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: should return an array of objects, each with properties (comment_id, votes, created_at, author, body, review_id)", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(3);
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              review_id: expect.toBeInteger(2),
+            })
+          );
+        });
+      });
+  });
+  test("200: returned array should be sorted by created_at, descending", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("404: not found if no comments with the specified review_id", () => {
+    return request(app)
+      .get("/api/reviews/200/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("not found");
+      });
+  });
+  test("400: bad request if review_id not valid data type", () => {
+    return request(app)
+      .get("/api/reviews/pineapple/comments")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("bad request");
+      });
+  });
+});
+
+describe('POST /api/reviews/:review_id/comments', () => {
+    test("201: responds with posted comment if succesful", () => {
+        const newComment = { username: 'dav3rid', body: 'life changing'}
+        return request(app)
+        .post("/api/reviews/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({body: {comment}}) => {
```

Comment:
Rather than using toEqual here, best practice would be to again use 'expect.objectContaining' to ensure our tests aren't brittle. 👍